go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/IBM/go-sdk-core/v5 v5.4.4
+	github.com/IBM/platform-services-go-sdk v0.18.18 // indirect
```

Comment:
We shouldn't depend on platform-services-go-sdk. I see you used it for composing the headers, please use github.com/IBM/scc-go-sdk/common instead.